tests/updates.rs

```diff
@@ -46,6 +47,89 @@ fn test_updates_and_merging() {
 
     // Sending
     let mut doc_send = Automerge::new();
+    let mut tx = doc_send.transaction();
+    let (player_id, camera_id, numbers_id) =
+        send(&mut tx, &player_send, &camera_send, &numbers_send);
+    let (commit, _patch_log) = tx.commit();
+    let base_commit = commit.unwrap();
+    // Send the whole document
+    let data = doc_send.save();
+
+    // Receiving
+    let mut doc_receive = Automerge::load(&data).unwrap();
+    let (mut player_receive, mut camera_receive, mut numbers_receive) = receive(&doc_receive);
+    assert_eq!(player_send, player_receive);
+    assert_eq!(camera_send, camera_receive);
+    assert_eq!(numbers_send, numbers_receive);
+
+    // Updating
+    let mut tx = doc_send.transaction();
+    update1(
+        &mut tx,
+        &mut player_send,
+        &mut camera_send,
+        &mut numbers_send,
+        &player_id,
+        &camera_id,
+        &numbers_id,
+    );
+    let (commit, _patch_log) =
+        tx.commit_with(CommitOptions::default().with_message("Updating position"));
+    let _commit_after_update1 = commit.unwrap();
+
+    assert_ne!(player_send, player_receive);
+
+    let data = doc_send.save_after(&[base_commit]);
+
+    // Update receiving side in the meantime too
+    let (player_id, camera_id, numbers_id) = (
+        doc_receive.get(ObjId::Root, PLAYER).unwrap().unwrap().1,
+        doc_receive.get(ObjId::Root, CAMERA).unwrap().unwrap().1,
+        doc_receive.get(ObjId::Root, NUMBERS).unwrap().unwrap().1,
+    );
+    update2(
+        &mut doc_receive,
+        &mut player_receive,
+        &mut camera_receive,
+        &mut numbers_receive,
+        &player_id,
+        &camera_id,
+        &numbers_id,
+    );
+    println!();
+
+    // Receiving
+    doc_receive.load_incremental(&data).unwrap();
+    let (player_receive2, camera_receive2, numbers_receive2) = receive(&doc_receive);
+
+    // Assert that local and remote changes were merged successfully
+    // XXX: Inline the functions, the code below implies knowing thwat update1() and update2() touched...
```

Comment:
```suggestion
    // XXX: Inline the functions, the code below implies knowing that update1() and update2() touched...
```
XXX -> TODO? Or not even relevant anymore?

Reply:
Nice typo :D